src/components/frame/Extensions/pages/ExtensionDetailsPage/ExtensionTopRow.tsx

```diff
@@ -65,8 +82,33 @@ const ExtensionsTopRow: FC<ExtensionsTopRowProps> = ({
       address: address || '',
     });
 
+  const handleUpgradeColony = useCallback(
+    () =>
+      toggleActionSidebarOn({
+        [ACTION_TYPE_FIELD_NAME]: Action.UpgradeColonyVersion,
+      }),
+    [toggleActionSidebarOn],
+  );
+
   return (
     <>
+      {(extensionData.neededColonyVersion || 0) > colony.version && (
+        <div className="pb-6">
+          <NotificationBanner
+            status="error"
+            icon={WarningCircle}
+            callToAction={
+              <button type="button" onClick={handleUpgradeColony}>
+                {formatText(MSG.upgradeColony)}
+              </button>
```

Comment:
At the moment, the underline is being overlapped by the bottom parts of the button copy:

| Current implementation | Figma design |
| ---- | ---- |
| ![Screenshot 2024-08-20 at 05 55 16](https://github.com/user-attachments/assets/c1b9ac6f-0da5-4801-a9f7-a7a06854f5fd) | ![Screenshot 2024-08-20 at 05 55 27](https://github.com/user-attachments/assets/4e8ee044-f7b2-4c90-bf69-d91d5aae3cbc) |

If you just add the following `callToActionClassName` value, it should be good to go!

```ts
<NotificationBanner
  status="error"
  icon={WarningCircle}
  callToAction={
    <button type="button" onClick={handleUpgradeColony}>
      {formatText(MSG.upgradeColony)}
    </button>
  }
  callToActionClassName="underline-offset-2" // <<-- Add me :)
>
  {formatText(MSG.colonyVersionTooLow, {
    minimumColonyVersion: extensionData.neededColonyVersion,
  })}
</NotificationBanner>
```

**Result**
![image](https://github.com/user-attachments/assets/228ba41a-9bf7-4816-a6ab-b2ea73e550bf)

Reply:
**Figma link:** https://www.figma.com/design/6qONTE1uD9K4yoIbfFzidg/Extensions?node-id=2187-15468&t=weswhiL8GdhTqFqW-4

Reply:
Nice catch, thanks @rumzledz. I've added it globally to the notification banner call to action after checking the designs, seems like it should always have an offset of 2px.